src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingGracefulShutdownSpec.cs

```diff
@@ -190,6 +190,7 @@ public void ClusterSharding_should_setup_shared_journal()
             Persistence.Persistence.Instance.Apply(Sys);
             RunOn(() =>
             {
+                throw new Exception("Exception of first node");
```

Comment:
Need to make sure we remove this before we merge in this PR. Was wondering why that test was failing when I was going through the Azure DevOps logs :p

Reply:
Sure - was keeping it there to be sure that we will get some failed tests with logs

Reply:
All good